helm/dex-app/templates/dex/secret.yaml

```diff
@@ -104,6 +105,11 @@ stringData:
       {{- include "trusted-peers" ( concat (.Values.oidc.staticClients.dexK8SAuthenticator.trustedPeers | default list ) (list .Values.oidc.staticClients.gsCLIAuth.clientID) ) }}
     {{- end }}
     {{- include "print-clean-extra-static-clients" .Values.oidc.extraStaticClients }}
+    staticPasswords:
+    - email: {{ .Values.oidc.staticPasswords.email }}
+      hash: {{ .Values.oidc.staticPasswords.hash }}
+      username: {{ .Values.oidc.staticPasswords.username }}
+      userID: {{ .Values.oidc.staticPasswords.userID }}
```

Comment:
Shouldn't we check if a password is given in values before creating this structure ?